pkgs/by-name/oo/ooklaserver/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, autoPatchelfHook
+}:
+let
+  filenameMapping = {
+    "x86_64-linux" = "OoklaServer-linux-x86_64-static-musl.zip";
+    # Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit executable x86_64] [arm64]
+    "x86_64-darwin" = "OoklaServer-macosx.zip";
+    "aarch64-darwin" = "OoklaServer-macosx.zip";
+    "aarch64-linux" = "OoklaServer-linux-aarch64-static-musl.zip";
+    "i686-windows" = "OoklaServer-windows32.zip";
+    "x86_64-windows" = "OoklaServer-windows64.zip";
+    # OoklaServer-linux64-deb9.zip
+    # OoklaServer-freebsd12_64.zip
+    # OoklaServer-freebsd13_64.zip
+  };
+in
+stdenv.mkDerivation rec {
+  version = "2.11.1.2";
+  pname = "OoklaServer";
```

Comment:
The `pname` attribute may not contain uppercase letters.

Reply:
I do see some pname with uppercase letters, is there a standard for this?

Reply:
See `pkgs/README.md`, we are in the process of normalizing these.